examples/sst2/train.py

```diff
@@ -18,20 +18,28 @@
 from typing import Any, Callable, Dict, Iterable, Mapping, Optional, Sequence, Tuple, Union
 
 from absl import logging
+import flax
 from flax import optim
 from flax.metrics import tensorboard
+from flax.training import train_state
 import input_pipeline
 import jax
 import jax.numpy as jnp
 import ml_collections
 import models
 import numpy as np
 import tensorflow as tf
+import optax
 
 Array = jnp.ndarray
 Example = Dict[str, Array]
 
 
+class TrainState(train_state.TrainState):
+  model_state: Any
+  dynamic_scale: Optional[flax.optim.DynamicScale]
```

Comment:
Why do you need a `dynamic_scale`? Check out the `imagenet` example for an application of that class.